models/users.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class users extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  users.init({
+    id: DataTypes.INTEGER,
+    username: DataTypes.STRING,
+    email: DataTypes.STRING,
+    avatar: DataTypes.BLOB
+  }, {
+    sequelize,
+    modelName: 'users',
+  });
+  return users;
+};
```

Comment:
nice clean code!

Reply:
thanks! @tomselden you should have a invite to become a contributor so that your approval counts towards the merge